sdk/src/openssl/ec_signer.rs

```diff
@@ -118,93 +115,6 @@ impl Signer for EcSigner {
     }
 }
 
-// C2PA use P1363 format for EC signatures so we must
-// convert from ASN.1 DER to IEEE P1363 format to verify.
-struct ECSigComps<'a> {
-    r: &'a [u8],
-    s: &'a [u8],
-}
-
-fn parse_ec_sig(data: &[u8]) -> der_parser::error::BerResult<ECSigComps> {
-    // IMPORTANT: OpenSslMutex::acquire() should have been called by calling fn.
-    // Please don't make this pub or pub(crate) without finding a way to ensure
-    // that precondition.
```

Comment:
Looks like this comment was lost in the move to `sig_utils.rs`. It calls out not making this function `pub(crate)` without finding a way to ensure the precondition that `OpenSslMutex::acquire()` should have been called first. Did you find a way to ensure that precondition?

Reply:
Perfect, sounds good